GraphsV2/AreaGraph/index.js

```diff
@@ -0,0 +1,116 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { compose } from 'redux';
+import { scaleOrdinal } from 'd3-scale';
+import { schemeCategory10 } from 'd3-scale-chromatic';
+import {
+    AreaChart,
+    Area,
+} from 'recharts';
+
+import WithConfigHOC from '../../HOC/WithConfigHOC';
+import WithValiddataHOC from '../../HOC/WithValidationHOC';
+import customTooltip from '../Components/utils/RechartsTooltip';
+import dataParser from '../utils/DataParser';
+import renderLegend from '../Components/utils/Legend';
+import config from './default.config';
+import xAxis from '../Components/utils/xAxis';
+import yAxis from '../Components/utils/yAxis';
+
+const AreaGraph = (props) => {
+
+    const {
+        properties,
+        height,
+        width,
+        data,
+    } = props;
+
+    const {
+        xLabel,
+        yLabel,
+        xColumn,
+        yColumn,
+        tooltip,
+        legend,
+        XAxisLabelConfig,
+        margin,
+        xLabelRotateHeight,
+        dateHistogram,
+        xTickFormat,
+        yTickFormat,
+        linesColumn,
+        stacked,
+    } = properties;
+
+    // Formatting data for direct consumption by Area Graph
+    const { parsedData, uniqueKeys: areaKeys } = dataParser({ 
+      data, 
+      key: linesColumn, 
+      xColumn, 
+      yColumn,
+    });
+
+    const legendHeight = (legend.separate * height) / 100;
```

Comment:
similar to PieGraph, the legendHeight should be calculated directly in renderLegend